crates/syntax/test_data/parser/validation/0045_ambiguous_trait_object.rs

```diff
@@ -1,6 +1,6 @@
-type Foo<'a> = &'a dyn Send + Sync;
-type Foo = *const dyn Send + Sync;
-type Foo = fn() -> dyn Send + 'static;
-fn main() {
-    let b = (&a) as &dyn Add<Other, Output = Addable> + Other;
-}
+type Foo<'a> = &'a dyn Send + Sync;
+type Foo = *const dyn Send + Sync;
+type Foo = fn() -> dyn Send + 'static;
+fn main() {
+    let b = (&a) as &dyn Add<Other, Output = Addable> + Other;
+}
```

Comment:
What is the difference in this file?

Reply:
Line endings I would assume?

Reply:
CR => LF (as for the other 2 files in the same folder), see https://github.com/rust-lang/rust-analyzer/pull/12871#discussion_r928817419